### PR TITLE
Add the man page installation

### PR DESCRIPTION
**Pull request by 0mp**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,6 +17,8 @@ setaudit:	$(OBJS)
 install:	setaudit
 		mkdir -p $(PREFIX)/bin/
 		install -m 0555 setaudit $(PREFIX)/bin/setaudit
+		mkdir -p $(PREFIX)/man/man1/
```
Comment (csjayp) [inline Makefile:20]: Do we need a: 
```shell
gzip -k setaudit.1
```
Here? It looks like everything in /usr/local/man/manX is compressed.

Comment (0mp) [reply Makefile:20]: From my experience the ports framework compresses man pages automagically behind the scenes. 

There's no need to do it, I think.

**Comment by csjayp** [comment]
OK -- I think that is fine. Thanks again!

**Comment by 0mp** [comment]
😈 

Ok, I'm preparing the port now.

**Pull request merged**